managed/services/telemetry/config.default.yml

```diff
@@ -67,57 +59,189 @@ telemetry:
+    summary: "Agents version POSTGRES_EXPORTER"
+    data:
+      - metric_name: "pmm_client_postgres_exporter_version"
+        column: "version"
+
+  - id: NodesInfoProxysqlExporter
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'proxysql-exporter' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version PROXYSQL_EXPORTER"
+    data:
+      - metric_name: "pmm_client_proxysql_exporter_version"
+        column: "version"
+
+  - id: NodesInfoQANMysqlPerfschemaAgent
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'qan-mysql-perfschema-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_MYSQL_PERFSCHEMA_AGENT"
+    data:
+      - metric_name: "pmm_client_qan_mysql_perfschema_agent_version"
+        column: "version"
+
+  - id: NodesInfoQANMysqlSlowlogAgent
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'qan-mysql-slowlog-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_MYSQL_SLOWLOG_AGENT"
+    data:
+      - metric_name: "pmm_client_qan_mysql_slowlog_agent_version"
+        column: "version"
+
+  - id: NodesInfoQANMongodbProfilerAgent
     source: PMMDB_SELECT
-    query: agent_type, version FROM agents
-    summary: "Agents version"
+    query: version FROM agents WHERE 'agent_type = 'qan-mongodb-profiler-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_MONGODB_PROFILER_AGENT"
     data:
-      - metric_name: "nodes_info_agent"
-        column: "agent_type"
-      - metric_name: "nodes_info_version"
+      - metric_name: "pmm_client_qan_mongodb_profiler_agent_version"
         column: "version"
-      - metric_name: "classifier"
-        column: "agent_type"
+
+  - id: NodesInfoQANPostgresqlPgstatementsAgent
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'qan-postgresql-pgstatements-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_POSTGRESQL_PGSTATEMENTS_AGENT"
+    data:
+      - metric_name: "pmm_client_qan_postgresql_pgstatements_agent_version"
+        column: "version"
+
+  - id: NodesInfoQANPostgresqlPgstatmonitorAgent
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'qan-postgresql-pgstatmonitor-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_POSTGRESQL_PGSTATMONITOR_AGENT"
+    data:
+      - metric_name: "pmm_client_qan_postgresql_pgstatmonitor_agent_version"
+        column: "version"
+
+  - id: NodesInfoRDSExporter
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'rds-exporter' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version RDS_EXPORTER"
+    data:
+      - metric_name: "pmm_client_rds_exporter_version"
+        column: "version"
+
+  - id: NodesInfoExternalExporter
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'external-exporter' and pmm_agent_id != 'pmm-server'
```

Comment:
```suggestion
    query: version FROM agents WHERE agent_type = 'external-exporter' and pmm_agent_id != 'pmm-server'
```

Reply:
as I remember external_exporter runs exactly on PMM Server, isn't it?

Reply:
no  External can run anywhere 

but!!!  for external exporter  we need it's  `group` label. This is  string and important one 

@ritbl ^

Reply:
@rnovikovP sorry, didn't get it. can you please explain.